web-console/1-homepage/catalog.md

```diff
@@ -36,9 +33,32 @@ On smaller viewport sizes, the catalog tabs should stack. ![hover](img/OpenShift
 - For secondary levels of categorization, there should again be an `Other` category if there are items to display (not shown in mockup).
 - Clicking on a subcategory will open a scrollable panel that contains the catalog items.
 
+### Responsive State
+![mobile-1](img/OpenShift-Next-Mobile-V2-scrolledup.png)
+
+- On smaller viewport sizes, the catalog tabs should stack with arrows to the right.
```

Comment:
@rhamilto, can you help clarify?  I'd like us to be more specific in the doc, to be sure that we all have the same understanding:
- Extra small devices (phones, less than 768px) 
- Small devices (tablets, 768px and up) 

If we say mobile viewport sizes, does that refer to extra small devices?

If so, @beanh66 let's change all references from "On smaller viewport sizes" to "On mobile"